.github/actions/validate-docs-links/src/checker.ts

```diff
@@ -74,11 +74,13 @@ const slugger = new GithubSlugger()
 async function getAllMdxFilePaths(basePath: string): Promise<RouteFragment[]> {
   const sidebarLearn: ConfigSchema = JSON.parse(await fs.readFile(path.join(basePath, 'config/sidebar-learn.json'), 'utf8'))
   const sidebarReference: ConfigSchema = JSON.parse(await fs.readFile(path.join(basePath, 'config/sidebar-reference.json'), 'utf8'))
+  const sidebarGuides: ConfigSchema = JSON.parse(await fs.readFile(path.join(basePath, 'config/sidebar-guides.json'), 'utf8'))
   const footer: FooterConfigSchema = JSON.parse(await fs.readFile(path.join(basePath, 'config/sidebar-footer.json'), 'utf8'))
 
   const config = [
     ...sidebarLearn.map(group => ({...group, slug: path.join('learn',  group.slug)})),
-    ...sidebarReference.map(group => ({...group, slug: path.join('reference/', group.slug)}))
+    ...sidebarReference.map(group => ({...group, slug: path.join('reference/', group.slug)})),
+    ...sidebarGuides.map(group => ({...group, slug: path.join('guides/', group.slug)}))
```

Comment:
More out of curiosity than anything: why do we use `learn` for the first item, but `reference/` and `guides/` for the others?

Reply:
No reason really, the `/` is ignored by the `path.join()` function AFAIK. Worth fixing but I didn't want to do too much editing beyond what's needed.